unlock-app/src/__tests__/components/interface/Header.test.js

```diff
@@ -19,3 +22,20 @@ describe('mapStateToProps', () => {
     expect(props).toHaveProperty('pathname', '/')
   })
 })
+
+describe('isOnAppPage', () => {
+  it('should return false if no app button matches the current pathname', () => {
+    expect.assertions(1)
+    expect(isOnAppPage('/about')).toEqual(false)
+  })
+
+  it('should return the pathnam if an app button equals the current pathname', () => {
+    expect.assertions(1)
+    expect(isOnAppPage('/dashboard')).toEqual('/dashboard')
```

Comment:
My only concern with this is that this kind of type punning will get weird when we start working with TypeScript